app/blueprints/pdf_form_blueprint.rb

```diff
@@ -0,0 +1,21 @@
+class PdfFormBlueprint < Blueprinter::Base
+  identifier :id
+
+  fields :form_type, :status, :created_at
+
+  field :form_json do |pdf_form, _options|
```

Comment:
This is redundant if it matches the public method. you only need

field :form_json